lib/asset/baseAsset.js

```diff
@@ -69,6 +72,41 @@ class BaseAsset {
         return this._asset_id;
     }
 
+    /**
+     * Set multiple fields at once.
+     * @param {Object} fields - Contains See each setter below for a
+     * description of each field.
+     */
+    set_fields(fields) {
+        if (fields.title) {
+            this.set_title(fields.title);
+        }
+        if (fields.synopsis) {
+            this.set_synopsis(fields.synopsis);
+        }
+        if (fields.thumbnail) {
+            this.set_thumbnail(fields.thumbnail);
+        }
+        if (fields.canonical_uri) {
+            this.set_canonical_uri(fields.canonical_uri);
+        }
+        if (fields.last_modified_date) {
+            this.set_last_modified_date(fields.last_modified_date);
+        }
+        if (fields.date_published) {
+            this.set_date_published(fields.date_published);
+        }
+        if (fields.sequence_number) {
```

Comment:
These should probably distinguish between "field not present" and "field set to falsey value". It might not matter that much for the strings, but `sequence_number` can legitimately be 0.

Reply:
Good catch!